src/common/importer/snowflake/importer.js

```diff
@@ -267,14 +267,14 @@ function cards(currentState, json, bookId) {
       { position: 0, title: i18n('Main Plot') },
       bookId
     )
-    const scenesByChapter = groupBy(sceneListNode['GContainer'], (scene) => {
-      const node = scene['GInteger'].find((n) => n['_attributes']['name'] == 'chapter')
+    const scenesByBeat = groupBy(sceneListNode['GContainer'], (scene) => {
+      const node = scene['GInteger'].find((n) => n['_attributes']['name'] == 'beat')
```

Comment:
this 'chapter' comes from the Snowflake file, so it needs to stay 'chapter'

Reply:
Oops!
I did look through the importer and exporter and missed that.